cmd/api/src/model/appcfg/flag.go

```diff
@@ -29,6 +30,7 @@ const (
 	FeatureReconciliation      = "reconciliation"
 	FeatureEntityPanelCaching  = "entity_panel_cache"
 	FeatureAdcs                = "adcs"
+	FeatureClearGraphData      = "clear-graph-data"
```

Comment:
This should probably use underscores to match other flags

Reply:
You're right! This has been updated as suggested.